src/KMonad/Keyboard/ComposeSeq.hs

```diff
@@ -178,8 +178,10 @@ ssComposed =
     , ("/ o"      , 'ø'     , "oslash")
     , ("` u"      , 'ù'     , "ugrave")
     , ("' u"      , 'ú'     , "uacute")
+    , ("' U"      , 'Ú'     , "Uacute")
     , ("^ u"      , 'û'     , "ucircumflex")
     , ("\" u"     , 'ü'     , "udiaeresis")
+    , ("\" U"     , 'Ü'     , "Udiaeresis")
```

Comment:
This already seems to be defined [here](https://github.com/kmonad/kmonad/blob/64aaba001a535ec4cd49525eb658823468d6d58d/src/KMonad/Keyboard/ComposeSeq.hs#L150)

Reply:
I removed it. It was missing in 0.4.1, but didn't realize it was already in `master`.

Reply:
ah yeah seems to have been added in https://github.com/kmonad/kmonad/commit/b7214c4bfa7ae2d6d8ea99fd991aeb3b951fdb5a

Reply:
@jdnavarro doesn't seem like it's been removed :)

Reply:
Sorry about that, there you go.

Reply:
Perfect, thanks!